packages/sdk-core/src/modules/breadcrumbs/BreadcrumbsManager.ts

```diff
@@ -63,11 +65,16 @@ export class BreadcrumbsManager implements BacktraceBreadcrumbs, BacktraceModule
 
     public get(): Record<string, number> {
         return {
-            [this.BREADCRUMB_ATTRIBUTE_NAME]: this.breadcrumbsStorage.lastBreadcrumbId,
+            [this.BREADCRUMB_ATTRIBUTE_NAME]: this._storage.lastBreadcrumbId,
         };
     }
 
-    public bind({ reportEvents }: BacktraceModuleBindData): void {
+    public bind({ client, reportEvents }: BacktraceModuleBindData): void {
+        for (const attachment of this._storage.getAttachments()) {
+            client.addAttachment(attachment);
+        }
+        client.addAttributeProvider(this);
```

Comment:
qq: the AddAttribute method won't work? I think you can just use this method and do not expose additional redundant method.

Reply:
so `client.add(() => {"last.breadcrumbid": this._lastBreadcrumbid});` won't work? it will create a dynamic attribute provider anyway. Less exposed public API - less things to maintain and less things that can break

Reply:
Hmm, I guess it will. I'll change that.